phoenix-pig/src/main/java/org/apache/phoenix/pig/util/PhoenixPigSchemaUtil.java

```diff
@@ -62,7 +62,7 @@ public static ResourceSchema getResourceSchema(final Configuration configuration
                 Preconditions.checkNotNull(sqlQuery, "No Sql Query exists within the configuration");
                 final SqlQueryToColumnInfoFunction function = new SqlQueryToColumnInfoFunction(configuration);
                 columns = function.apply(sqlQuery);
-            } else {
+            } else if (SchemaType.TABLE.equals(schemaType)) {
```

Comment:
nit: Earlier you used == for enum comparison, is there a standard for Phoenix?

Reply:
Yes I agree that we should standardize it. There is no pre-defined standard here and both of them work in theory. Any particular prefs you suggest for one over other? Also can you point me to other location (if you remember)?

Reply:
I prefer == as you won't accidentally order the equality wrong and risk a NPE.